README.md

```diff
@@ -1,2 +1,6 @@
 # openmcyclus
-Repository of Cyclus archetypes to couple Cyclus with OpenMc
+Repository of Cyclus archetypes to couple Cyclus with OpenMC
```

Comment:
```suggestion
Repository of [Cyclus](https://fuelcycle.org/) archetypes to couple Cyclus with [OpenMC](https://docs.openmc.org/en/develop/pythonapi/generated/openmc.run.html)
```

Reply:
I was planning on adding this information in a later PR, but adding it now is fine.